ui/src/cohortReview/newReviewDialog.tsx

```diff
@@ -47,15 +47,17 @@ export type NewReviewDialogProps = {
 } & UseNewReviewDialogProps;
 
 export function NewReviewDialog(props: NewReviewDialogProps) {
-  const source = useSource();
+  const underlaySource = useUnderlaySource();
 
   const countState = useSWRImmutable(
     {
       type: "count",
       cohort: props.cohort,
     },
     async () => {
-      return await source.filterCount(generateCohortFilter(props.cohort));
+      return await underlaySource.filterCount(
+        generateCohortFilter(props.cohort)
+      );
```

Comment:
I see several of these formatting changes. Did you run some command to do this automatically? If so, is there a way to make that happen for every change? For the Java code, there's a Gradle plugin and the GHAs run a check (also via the plugin, not my code) that fails if it hasn't been formatted. Maybe we can do something similar for the UI code? (Not for this PR, just noticed it here)

Reply:
Sounds good. The generated docs + ts file is another pattern -- runs a command then does a `git diff` and fails if anything changed. You could add it there, too.